capstone_project/src/main/webapp/pages/resources.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+<script src="../scripts/comments.js"></script>
+<link rel="stylesheet" href="../styles/posts.css">
+<body onload="loadResources()">
+  <div class="row">
+    <div class="col s3" id="sidebar"></div>
+    <div class="col s6">
+      <div class="center" id="header">
+        <h3>View All Resources Here</h3>
+      </div>
+      <div id="resources-display" data-sort="default">
+        <div class="resource">
+          <h4 class="category-name">General</h4>
+          <p class="resource-name"><a href="https://www.happify.com/">Happify</a></p>
+        </div>
+        <div class="resource">
+          <h4 class="category-name">Anxiety &amp; Stress</h4>
+          <p class="resource-name"><a href="https://www.nami.org/help">National Alliance on Mental Health</a></p>
+        </div>
+        <div class="resource">
+          <h4 class="category-name">Depression</h4>
+          <p class="resource-name"><a href="https://www.crisistextline.org/">Crisis Textline</a></p>
+        </div>
+        <div class="resource">
+          <h4 class="category-name">Troubled Relationships</h4>
+          <p class="resource-name"><a href="https://www.loveisrespect.org/">Love is Respect</a></p>
+        </div>
+      </div>
+    </div>
+    <div class="col s3" id="add-resource-container">
+      <form id="post-input"
+            method="POST"
+            onsubmit="disableInput(event)">
+        <div  class="input-field">
```

Comment:
nit: there should only be one space between "div" and "class"

Reply:
+1